graql/reasoner/concept-inequality.feature

```diff
@@ -572,15 +569,12 @@ Feature: Concept Inequality Resolution
     Then for graql query
       """
       match
+        $value isa! retailer;
+        $unwantedValue isa! retailer;
         $x has $value;
         $y has $unwantedValue;
-        not { $value is $unwantedValue; };
         $unwantedValue "Ocado";
-        $value isa! $type;
-        $unwantedValue isa! $type;
-        $type type base-attribute;
-        not { $type is $unwantedType; };
-        $unwantedType type string-attribute;
+        not { $value is $unwantedValue; };
```

Comment:
According to the test name we're restricting type and attribute value separately, so it follows that this should be `$value != $unwantedValue;`

Reply:
addressed